Code/GraphMol/MolDraw2D/MolDraw2D.cpp

```diff
@@ -1068,6 +1069,10 @@ pair<string, MolDraw2D::OrientType> MolDraw2D::getAtomSymbolAndOrientation(
     symbol = "";
   } else if (isComplexQuery(&atom)) {
     symbol = "?";
+  } else if (drawOptions().atomLabelDeuteriumTritium && atom.getAtomicNum() == 1 &&
```

Comment:
I tend to think it would be nicer to have something like:

```
symbol = drawOptions().getAtomLabel(atom);
```

This allows users to perhaps override logic and makes the system more extensible for different purposes.